source/support/topics/enclave/how-do-i-setup-an-apex-redirect-using-aws.md

```diff
@@ -0,0 +1,149 @@
+This tutorial will guide you through the process of setting up an Apex redirect
+using AWS S3, AWS CloudFront, and AWS Certificate Manager.
+
+The heavy lifting is automated using CloudFormation, so this entire process
+shouldn't require more than a few minutes of active work.
+
+Before starting, you will need the following:
+
+- The domain you want to redirect away from (e.g.: `example.com`, `myapp.io`,
+  etc.).
+- The subdomain you want to redirect to (e.g.: `app`, `www`, etc.).
+- Access to the DNS configuration for the domain. Your DNS provider must
+  support ALIAS records (also known as CNAME flattening). We support the
+  following DNS providers in this tutorial: Amazon Route 53, CloudFlare,
+  DNSimple. If your DNS provider does not support ALIAS records, then we
+  encourage you to migrate your NS records to one that does.
+- Access to one of the mailboxes used by AWS Certificate Manager to validate
+  ownership of your domain. If you registered the domain yourself, that should
+  be the case, but otherwise [review the relevant AWS Certificate Manager
+  documentation][0] first.
+- An AWS account.
+
+After completing this tutorial, you will have an inexpensive highly-available
+redirect from your domain apex to your subdomain, which will require absolutely
+no maintenance going forward.
+
+## Create the CloudFormation Stack
+
+Navigate to [the CloudFormation Dashboard][10], and click "Create Stack".
+
+Choose "Specify an Amazon S3 template URL", and use the following template URL:
+`https://s3.amazonaws.com/www.aptible.com/assets/cloudformation-redirect.json`.
+
+Click "Next", then:
+
+- For the `Stack name`, choose any name you'll recognize in the future, e.g.:
+  `redirect-example-com`.
+- For the `Domain` parameter, input the domain you want to redirect away from.
+- For the `Subdomain` parameter, use the subdomain. Don't include the domain
+  itself there! If you want to redirect to `app.example.com`, then just input
+  `app`.
+- For the `ViewerBucketName` parameter, input any name you'll recognize in the
+  future. You cannot use dots here. A name like `redirect-example-com` will
+  work here too.
+
+Then, hit "Next", and click through the following screen as well.
+
+
+## Validate Domain Ownership
+
+In order to set up the apex redirect to require no maintenance, the
+CloudFormation template we provide uses AWS Certificate Manager to
+automatically provision (and renew) a certificate to serve the redirect from
+your domain apex to your subdomain.
+
+To make this work, you'll need to validate with AWS that you own the domain
+you're using. So, once the CloudFormation stack enters the state
+`CREATE_IN_PROGRESS`, navigate to your mailbox, and look for an email from AWS
+to validate your domain ownership.
+
+Once you receive it, read the instructions and click through to validate.
+
+And don't worry: customers will never actually see that certificate (their
```

Comment:
Hmm; I'm inclined to disagree (otherwise I wouldn't have put it there 😄), but I think it's fair to remove it, and see if we ever get asked about this. If we do, we can always put it back!